Firebase/Database/Login/FAuthTokenProvider.m

```diff
@@ -15,27 +15,31 @@
  */
 
 #import "FAuthTokenProvider.h"
-#import "FUtilities.h"
+
+#import <FirebaseAuthInterop/FIRAuthInterop.h>
 #import <FirebaseCore/FIRAppInternal.h>
+#import <FirebaseCore/FIRComponent.h>
```

Comment:
I believe this can be removed, along with the App import, only keeping the AuthInterop import.

Reply:
Actually, the AppInternal is needed for the `FIRAuthStateDidChangeInternalNotification` declaration.

Reply:
Actually, the AppInternal is needed for the